pkg/auth/audit/audit.go

```diff
@@ -208,3 +212,56 @@ func isExist(array []string, key string) bool {
 	}
 	return false
 }
+
+func (a *auditLog) concealSensitiveData(requestURI string, body []byte) []byte {
+	var m map[string]interface{}
+	if err := json.Unmarshal(body, &m); err != nil {
+		return body
+	}
+
+	var changed bool
```

Comment:
would be good to add comment that this block for secrets and below `conceal...` is for non-secrets

Reply:
Done.